src/main/java/uk/gov/hmcts/probate/controller/DocumentController.java

```diff
@@ -169,15 +140,24 @@
             documentService.expire(callbackRequest, documentType);
         }
 
+        List<Document> documents = new ArrayList<>();
```

Comment:
I would like to minimise the logic in the controllers if possible. Maybe split out the specific doc generation paths into some other services?